about.html

```diff
@@ -0,0 +1,159 @@
+      <h1 class="co red headp"> Web programming courses for creative and busy women 2023 </h1>
+    </div>
+    <div class="para1 co">
+      <P> 
+        The Global summit is an international nonprofit dedicated to inspiring women to excel in technology careers.#women who code is here to help bridge the gender gap in the tech programming industry by providing knowledge that is becoming necessary in today’s increasingly technological job market.
+        The Summit brings together the community of exeperts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two. This year, we hope to expand our invitation list including organizations and individuals.
+      </P>
+    </div>
+    <div class="para2 la">
+           <p class=" para2-first la"> Please contact us per Email for any further questions about web development 2023  </p>
+            <p class="second la"><strong><a href="#">beteland23@gmail.com</a></strong></p>
+    </div>
+   
+   </div>
+   </section>
+   <section class="container-back">
+   <section class="container la">
+    <p class="strong"><strong> Creative Common Global summit 2015 logo</strong>  </p>
+    <hr>
+    <p class="light"> The logo of 2015 world of women coders is desided form the logo compition from 23 june to 30.
+    </p>
+    <div class="image">
+        <img src="img/logo.PNG" alt="logo">
+    </div>
+   </section>
+  </section>
+   <section class="past-summit">
+    <p class="str la"><strong> See the past cc Global Summit</strong>
+      </p>
+      <hr>
+    <p class="lig la none"> take look at the last two CC gloabla summit which took place </p>
+    <p class="la cut none"> in Bueno sares and Addis Ababa</p>
+    <p class="one la">take look at the last two CC gloabla summit which took place in Bueno sares and Addis Ababa</p>
+    <section class="history">
+        <div class="first-histo">
+            <strong class="year la">2022</strong>
+            <p class="la cc">CC Global summit 2022 in Addis Ababa</p>
+        </div>
+        <div class="sec-histo">
+            <strong class="year la">2022</strong>
+            <p class="la cc">CC Global summit 2022 in Bueno sares </p>
+        </div>
+    </section>
+   </section>
+   <footer>
+    <div class="footer-first2">
+        <h3>Partner</h3>
+        <hr class="last-hr">
+        <div class="sponsor2">
+            <a href="#"><img src="img/mozila.png" alt="mozilla"></a>
+            <a href="#"><img src="img/google.png" alt="google"></a>
+            <a href="#"><img src="img/naver.PNG" alt="naver"></a>
+            <a href="#"><img src="img/daum.png" alt="daumkakao"></a>
+            <a href="#"><img src="img/air.png" alt="airbnb"></a>
+        </div>
+    </div>
+    <div class="footer-second2">
+        <img src="img/mobil-foot.PNG" alt="" class="footer-img">
+        <p>2023 Creative Commons Korea. Some Rights Reserved.</p>
+    </div>
+  </footer>
```

Comment:
- [ ] Kindly make sure the about footer match the provided design and remove the partner section. 
    - [ ] Please, The partner section should not be present in the about page, consider removing it.
    - [ ] Please, update the footer, make the background dark and the text white. as it's in the provided design.

**your project**
![image](https://user-images.githubusercontent.com/74411135/235254172-d3f24904-370a-4785-b732-0c4dc7ccac1b.png)

**provided design**
![image](https://user-images.githubusercontent.com/74411135/235254239-7c96f8ce-bd5f-4075-ac07-203fa716342c.png)

This will make your project way more prettier which will improve the user experience 📱  😄 👍